src/test_lib_json/CMakeLists.txt

```diff
@@ -1,39 +1,29 @@
 # vim: et ts=4 sts=4 sw=4 tw=0
 
-add_executable(jsoncpp_test
-    jsontest.cpp
-    jsontest.h
-    fuzz.cpp
-    fuzz.h
-    main.cpp
-)
+ADD_EXECUTABLE(jsoncpp_test
+        jsontest.cpp
+        jsontest.h
+        fuzz.cpp
+        fuzz.h
+        main.cpp
+        )
 
-
-if(BUILD_SHARED_LIBS)
-    if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.12.0)
-        add_compile_definitions( JSON_DLL )
-    else()
-        add_definitions( -DJSON_DLL )
```

Comment:
Can the compile time defines be removed?  This seems like a possible regression in behavior here.

Reply:
No, it does not produce regression, the definitions have already been stated [here](https://github.com/open-source-parsers/jsoncpp/pull/1374/files#diff-8cbbdca03fbe3b64d7024e4899f4abbc6f77db78837901951a5622918504fe79R115).